### ISSUE TITLE
L1: Limit transactions message byte-size

### ISSUE THREAD

**Issue by fkrause98** [lambdaclass/lambda_ethereum_rust#1132]
## Issue

As of writing this, there's an upper limit to how many transactions can be received in a single P2P message (256),
we should also have a limit on how big the sent and received transactions messages should be, based on their byte-size.

Check `ethereum_rust/crates/networking/p2p/rlpx/eth/transactions.rs` for more context.

**Comment by mpaulucci** [comment]
@mechanix97 do we have this implemented already? If so, close this ticket

**Comment by Mechanix97** [comment]
Yes, the limit was introduced in #3031 using the same number as geth
